java-backend/src/main/java/org/kframework/backend/java/symbolic/SymbolicRewriter.java

```diff
@@ -627,8 +628,11 @@ private K disjunctResults(List<K> results) {
             }
 
             for (ConstrainedTerm term : queue) {
+                boolean alreadyLogged = false;
+                try { //not formatting to minimize git merge conflicts.
```

Comment:
uhhhhhhh... this is what a git merge strategy that ignores whitespace is for.

Reply:
I did this to simplify code review mostly. Intellij knows to make diff modulo whitespaces. Can make another PR just for formatting. Are you ok with leaving as is?

Reply:
don't worry about code review, that's what ?w=1 is for.